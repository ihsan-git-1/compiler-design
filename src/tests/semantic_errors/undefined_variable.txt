class test {
    int x=2;

    int z = x+t;


  }