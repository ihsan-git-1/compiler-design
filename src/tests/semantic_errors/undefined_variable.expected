Undefined name t at (4,15)
